systemtests/src/test/java/io/enmasse/systemtest/isolated/monitoring/MonitoringTest.java

```diff
@@ -47,14 +48,26 @@
 
     @BeforeAll
     void installMonitoring() {
-        try { //TODO remove it after upgrade to surefire plugin 3.0.0-M5
-            EnmasseOperatorManager.getInstance().installMonitoringOperator();
-            Endpoint prometheusEndpoint = Kubernetes.getInstance().getExternalEndpoint("prometheus-route", environment.getMonitoringNamespace());
-            monitoring = new MonitoringClient(prometheusEndpoint);
-            monitoring.waitUntilPrometheusReady();
-            kubernetes.createNamespace(testNamespace);
-        } catch (Exception ex) {
-            beforeAllException = ex;
+        // If CRC remove exiting monitoring before and install app-monitoring-operator
+        if (Kubernetes.isCRC()) {
+            try {
+                uninstallMonitoring();
+            } catch (Exception e) {
+                beforeAllException = e;
+            }
+
+        }
+
+        if (!Kubernetes.isOpenShiftCompatible(OCP4) || Kubernetes.isCRC()) {
+            try { //TODO remove it after upgrade to surefire plugin 3.0.0-M5
+                EnmasseOperatorManager.getInstance().installMonitoringOperator();
+                Endpoint prometheusEndpoint = Kubernetes.getInstance().getExternalEndpoint("prometheus-route", environment.getMonitoringNamespace());
+                monitoring = new MonitoringClient(prometheusEndpoint);
+                monitoring.waitUntilPrometheusReady();
+                kubernetes.createNamespace(testNamespace);
```

Comment:
This line must run in every case, so move it to the beginning of this beforeall method.